Lib/socket.py

```diff
@@ -49,7 +51,7 @@
 import _socket
 from _socket import *
 
-import os, sys, io, selectors
+import os, sys, io, selectors, array
```

Comment:
I would prefer to only import array in functions using it: in recv_fds() and send_fds().